index.html

```diff
@@ -1,3 +1,29 @@
-<html>
-  <h1>hello main</h1>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <header class="header">
+        <h1>
+            My Logo
+        </h1>
```

Comment:
- [ ] The logo (if the design has one) is on the left nice however it has a link (a tag), please refactor with a tag